tutorials/text/annotations.py

```diff
@@ -104,6 +112,23 @@
 # Advanced Annotations
 # --------------------
 #
+# Placing Annotations Relative to Data
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Annotations can be positioned at a relative offset to the *xy* input to
+# annotation by setting the *textcoords* kwarg to one of the offset options.
+# In this example, the annotations are offset 1.5 points from the given *xy*
```

Comment:
Maybe define "points" here (or link out?) because folks often don't know what points are in this context  otherwise this looks good to me

Reply:
I agree with you but not sure what to do as  a few lines up is this line:

>  For physical coordinate systems (points or pixels) the origin is the bottom-left of the figure or axes. 

but neither one is clearly defined in https://matplotlib.org/devdocs/tutorials/advanced/transforms_tutorial.html#plotting-in-physical-coordinates which would be the I think obvious place to define them - and I've now found multiple definitions of both and so not sure which one we're using. The closest I get is 

> (In typography, a [point](https://en.wikipedia.org/wiki/Point_%28typography%29) is 1/72 inches, and by specifying your offsets in points, your figure will look the same regardless of the dpi resolution it is saved in.

from that tutorial